src/multibody/model.hxx

```diff
@@ -72,20 +72,11 @@ namespace se3
     nq += joint_model.nq();
     nv += joint_model.nv();
 
-    if (D::NV == Eigen::Dynamic)
-    {
-      effortLimit.conservativeResize(nv);effortLimit.bottomRows(joint_model.nv()) = max_effort;
-      velocityLimit.conservativeResize(nv);velocityLimit.bottomRows(joint_model.nv()) = max_velocity;
-      lowerPositionLimit.conservativeResize(nq);lowerPositionLimit.bottomRows(joint_model.nq()) = min_config;
-      upperPositionLimit.conservativeResize(nq);upperPositionLimit.bottomRows(joint_model.nq()) = max_config;
-    }
-    else
-    {
-      effortLimit.conservativeResize(nv);effortLimit.bottomRows<D::NV>() = max_effort;
-      velocityLimit.conservativeResize(nv);velocityLimit.bottomRows<D::NV>() = max_velocity;
-      lowerPositionLimit.conservativeResize(nq);lowerPositionLimit.bottomRows<D::NQ>() = min_config;
-      upperPositionLimit.conservativeResize(nq);upperPositionLimit.bottomRows<D::NQ>() = max_config;
-    }
+
+    effortLimit.conservativeResize(nv);effortLimit.bottomRows<D::NV>(joint_model.nv()) = max_effort;
```

Comment:
replace by
// Optimal efficiency here would be using the static-dim bottomRows<D::NV>, while specifying the dimension in argument in the case where D::NV is Eigen::Dynamic. However, this option is not compiling in Travis (why?). 
// As efficiency of Model::addJoint is not critical, the dynamic bottomRows is used here.
velocityLimit.conservativeResize(nv);velocityLimit.bottomRows(joint_model.nv()) = max_velocity;